plugins/common/opcua/input/input_client.go

```diff
@@ -318,6 +334,9 @@ func (o *OpcUAInputClient) InitNodeMetricMapping() error {
 			if node.IdentifierType == "" {
 				node.IdentifierType = group.IdentifierType
 			}
+			if node.MonitoringParams.SamplingInterval == 0 {
+				node.MonitoringParams.SamplingInterval = group.SamplingInterval
```

Comment:
ok if a node does not have an interval set, we default to the group's sample interval. What is the group's interval default? 

I ask as I want to ensure the current behavior is maintained and understand the defaults.

Reply:
See previous comment, I would suggest setting it to 0 to not break anything.